router/batchrouter/handle_async.go

```diff
@@ -476,6 +476,16 @@ func (brt *Handle) sendJobsToStorage(batchJobs BatchedJobs) {
 
 	_, ok := brt.asyncDestinationStruct[destinationID]
 	if ok {
+		if invalidManager, ok := brt.asyncDestinationStruct[destinationID].Manager.(*asynccommon.InvalidManager); ok {
+			failedAsyncJobs := BatchedJobs{
+				Jobs:       batchJobs.Jobs,
+				Connection: batchJobs.Connection,
+				TimeWindow: batchJobs.TimeWindow,
+				JobState:   jobsdb.Aborted.State,
+			}
+			brt.updateJobStatus(&failedAsyncJobs, false, invalidManager.Error, false)
+			return
+		}
```

Comment:
```suggestion
		if invalidManager, ok := brt.asyncDestinationStruct[destinationID].Manager.(*asynccommon.InvalidManager); !ok {
			failedAsyncJobs := BatchedJobs{
				Jobs:       batchJobs.Jobs,
				Connection: batchJobs.Connection,
				TimeWindow: batchJobs.TimeWindow,
				JobState:   jobsdb.Aborted.State,
			}
			brt.updateJobStatus(&failedAsyncJobs, false, invalidManager.Error, false)
			return
		}
```

Here we are  are handling error case sho shouldnt the condition be `!ok`?

Reply:
we are checking if manager is invalid manager and aborting jobs if so that means ok == true then we need to fail jobs.